web/docs/Backend Starters/Decoupled WordPress/configuring-build-hooks.md

```diff
@@ -0,0 +1,57 @@
+---
+id: 'configuring-build-hooks'
+title: 'Configuring Build Hooks'
+slug: '/backend-starters/decoupled-wordpress/configuring-build-hooks'
+sidebar_position: 2
+---
+
+For a decoupled technology stack, build hooks bring back a way to kick off and
+provide updates to a user’s site that is often lost when moving away from
+tightly coupled ecosystems. This feature allow user's to quickly trigger new
+builds of a site when updates are made on their blog, CMS or other data source.
+With build hooks, when a configured source of data is updated a deployment is
+triggered for it's corresponding front-end site.
+
+## Install and Activate WP Webhooks Plugin
+
+- Install and activate the
+  [WP-Webhooks Plugin](https://wordpress.org/plugins/wp-webhooks/).
+
+## Creating a Build Hook That Triggers When a Post Is Created Or Updated.
+
+First, navigate to your Pantheon Dashboard and select the site you would like to
+generate a build hook for. Open your site's settings, under the **Builds** tab,
+click **Add Build Hook**. Name your hook and select a branch to target using the
+**Branch to Build** dropdown. Copy the generated build hook URL.
+
+Next, navigate to your WordPress development site and open the WP Webhooks
+plugin dashboard, click on the **Send Data** tab. Select the **Post Created**
+Webhook trigger. Click **Add Webhook URL**, name the hook and paste the build
+hook URL that you generated earlier, submit this form. Repeat this process for
+the **Post Updated** trigger.
```

Comment:
Wondering if this section is better as a numbered list?

1. Navigate to the Pantheon Dashboard
2. Open the site's settings
3. etc...